3b.nextflow_production/scripts/manifest_io.py

```
@@ -0,0 +1,155 @@
+
+def load_manifest(path: str | Path) -> dict[str, Any]:
+    """Load the small pilot manifest from YAML or JSON."""
+    path = Path(path)
+    text = path.read_text()
+    try:
+        import yaml
+
+        loaded = yaml.safe_load(text)
+        return loaded or {}
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)